lib/web/assistant.go

```diff
@@ -371,7 +371,23 @@ func runAssistant(h *Handler, w http.ResponseWriter, r *http.Request,
 		h.log.WithError(err).Error("Error setting websocket readline")
 		return nil
 	}
-	defer ws.Close()
+	defer func() {
+		closureReason := websocket.CloseNormalClosure
+		closureMsg := ""
+		if err != nil {
+			_ = ws.WriteJSON(&assistantMessage{
+				Type:        assist.MessageKindError,
+				Payload:     err.Error(),
+				CreatedTime: time.Now().Format(time.RFC3339),
```

Comment:
I agree with the idea, but I don't think that converting to UTC should be a part of this PR. I need to double-check how UI handles the time. Otherwise, we will display the wrong timestamps.

Reply:
I checked all places and looks like we're using UTC everywhere else. I updated the code.